.circleci/config.yml

```diff
@@ -102,7 +102,6 @@ jobs:
                 echo "Changes in last commit do not need CI. Skipping step"
                 circleci-agent step halt
             fi
-      - run: ./scripts/format.sh && ./scripts/error_on_dirty.sh
```

Comment:
by not doing this we waste CI hours. i had this here so that builds would fail fast rather than having you find out they all passed except the lint one. i suppose we could make a separate ci job to do linting first that the other ones wait on to avoid this? but i thought it was cheap enough we could just have them all do it

Reply:
ah right, makes sense. it is cheap enough, I'll revert